docs/installing-vyper.rst

```diff
@@ -225,10 +225,16 @@ and try compiling a contract:
 Snap
 ****
 
-Vyper is published in the snap store. In any of the `supported Linux distros <https://snapcraft.io/docs/core/install>`_, install it with:
+Vyper is published in the snap store. In any of the `supported Linux distros <https://snapcraft.io/docs/core/install>`_, install it with (Note that installing the above snap is the latest master):
 ::
 
     sudo snap install vyper --edge --devmode
 
+::
+
+
+To install the latest beta version use:
+
+::
 
-(Note that installing the above snap is the latest master)
+    sudo snap install vyper --beta --devmode
```

Comment:
Can this install instructions be another PR?

Reply:
done